frontend/app/routes/project/show/community-facilities.js

```diff
@@ -2,6 +2,8 @@ import Route from '@ember/routing/route';
 import RSVP from 'rsvp';
 
 export default class ProjectShowCommunityFacilitiesRoute extends Route {
+  controllerName = 'project';
```

Comment:
What's this for?

Reply:
The `project` controller implements a method that hides the analysis step links if the user has `view_only` access to the project.

https://github.com/NYCPlanning/ceqr-app/blob/606b29df5eabf92200a25c96092b988343b822f1/frontend/app/controllers/project.js#L15-L17